test/distributions/test_multivariate_normal.py

```diff
@@ -296,6 +296,18 @@ def test_getitem(self):
         assert torch.equal(d.mean, dist.mean[1, 2, 2, :])
         self.assertAllClose(d.covariance_matrix, dist_cov[1, 2, 2, :, :])
 
+    def test_base_sample_shape(self):
```

Comment:
shoudl probably add explicit checks for the property for all the cases of `lazy_covariance_matrix`.